include/z64.h

```diff
@@ -867,8 +867,8 @@ typedef struct GameState {
     /* 0x00 */ GraphicsContext* gfxCtx;
     /* 0x04 */ GameStateFunc main;
     /* 0x08 */ GameStateFunc destroy;
-    /* 0x0C */ GameStateFunc nextGameStateInit;
-    /* 0x10 */ size_t nextGameStateSize;
+    /* 0x0C */ GameStateFunc init; // Current init, though after stopping, the graph thread will look here to determine the next gamestate to load
```

Comment:
```suggestion
    /* 0x0C */ GameStateFunc init; // Usually the current state's init, though after stopping, the graph thread will look here to determine the next gamestate to load.
```
Perhaps?

Reply:
Seems reasonable.

Reply:
I feel like it should probably use either `state` or `gameState consistently, though. `state` still seems a bit ambiguous.

Reply:
I agree, so went with `current gamestate's init`